notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    print(treeClassifier.score(test_x, test_y))
</pre><p>This looks good, you might consider printing a few other metrics, like precision/recall/f1-score etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Should I test with other, more complex classifiers? metrics seemed pretty solid so I didn't know if we should just run with this or try other things.

Reply:
Simple is good, but we can certainly have any number of simple models if we want to try them. Gradient boosted decision trees are about as complex as you want to go, I'd say logistic regression is something we'd also want to include for sure. If you do use a more complex model, try out a simple grid search over parameters.

Reply:
added logistic regression and a grid search on xgboostclassifier... the metrics on one of the species looks really poor probably due to a lack of training data and the logistic regression seems to perform the best so far